cli/ioctl/cmd/config/config_get_endpoint.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package config
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// configGetEndpointCmd represents the config get endpoint command
+var configGetEndpointCmd = &cobra.Command{
+	Use:       "get",
+	Short:     "Get endpoint for ioctl client",
+	ValidArgs: []string{"get", "endpoint"},
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return fmt.Errorf("accepts 1 arg(s), received %d", len(args))
+		}
+		if err := cobra.OnlyValidArgs(cmd, args); err != nil {
+			return err
+		}
+		return nil
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println(configGetEndpoint())
+	},
+}
+
+func init() {
+	ConfigCmd.AddCommand(configGetEndpointCmd)
+}
+
+func configGetEndpoint() string {
+	file, err := os.Open(configFileName)
+	if err != nil {
+		return fmt.Sprintf("failed to open config file %s", configFileName)
+	}
+	defer file.Close()
+
+	var endpoint string
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		// TODO: process other config lines like wallet
```

Comment:
Let's make things simple. If this line starts with "endpoint", we think this is the endpoint config. Implement this and remove TODO